src/Workspaces/Core/Portable/Storage/SQLite/v1/Interop/SafeSqliteBlobHandle.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using SQLitePCL;
+
+namespace Microsoft.CodeAnalysis.SQLite.v1.Interop
+{
+    internal sealed class SafeSqliteBlobHandle : SafeSqliteChildHandle<sqlite3_blob>
+    {
+        public SafeSqliteBlobHandle(SafeSqliteHandle sqliteHandle, sqlite3_blob? wrapper)
+            : base(sqliteHandle, wrapper?.ptr ?? IntPtr.Zero, wrapper)
+        {
+        }
+
+        protected override bool ReleaseChildHandle()
+        {
+            raw.sqlite3_blob_close(Wrapper);
+            return true;
```

Comment:
1. so, do all subclasses return true?  if so, is there value in teh bool?
2. should we be looking at the return values of the native method we're calling?

Reply:
> so, do all subclasses return true?

Currently, yes. `false` can be returned to indicate an error which is reported by an MDA during debugging.

> if so, is there value in teh bool?

I kept it as close to `ReleaseHandle` as possible (same return type with the same meaning).

> should we be looking at the return values of the native method we're calling?

If one of the return values means error, we could return `false` for that case. I'm neutral on this; the return value isn't used for anything aside from the debugger.